content/docs/user-guide/merge-conflicts.md

```diff
@@ -0,0 +1,173 @@
+# Merge Conflicts
+
+Sometimes multiple members of a team might want to edit the same file/directory
+that is tracked by DVC. And when the time comes to merge their changes together
+one will run into a merge conflict.
+
+## dvc.yaml
+
+This one is no different from your regular merge conflicts in code.
+
+```yaml
+stages:
+  prepare:
+    cmd: python src/prepare.py data/data.xml
+    deps:
+<<<<<<< HEAD
+    - data/big.xml
+=======
+    - data/small.xml
+>>>>>>> branch
```

Comment:
Won't these Git merge confuse IDEs though?

Reply:
These are normal git conflict markers. I'm not aware of any IDE that might be confused by it, but even if there are some, DVC is not the one creating it.

Reply:
Yeah VSC at least thinks this is an actual conflict...

![image](https://user-images.githubusercontent.com/1477535/90541565-77154900-e148-11ea-8859-937654c345ca.png)

I may change it a little... ⌛